src/pybind/mgr/progress/module.py

```diff
@@ -16,9 +16,7 @@
 import json
 
 
-ENCODING_VERSION = 2
-
-# keep a global reference to the module so we can use it from Event methods
+ENCODING_VERSIglobal reference to the module so we can use it from Event methods
```

Comment:
i am not sure this is a valid python statement.

Reply:
woops!